sdks/python/apache_beam/runners/portability/fn_api_runner.py

```diff
@@ -1139,7 +1139,7 @@ def __init__(self, state, provision_info, max_workers):
     self.provision_info = provision_info
     self.max_workers = max_workers
     self.control_server = grpc.server(
-        futures.ThreadPoolExecutor(max_workers=self.max_workers))
```

Comment:
I wonder if we can get rid of `max_workers` altogether here (and below).

Reply:
I left it here since it is computed based upon the number of environments.

Reply:
Yes, but we can simplify the code by omitting the computation altogether by making this (virtually) unbounded instead.